__tests__/storage/storage.test.js

```diff
@@ -85,12 +87,12 @@ describe('handleStop', () => {
     expect(nanos[0]).toEqual(testNano);
 
     // Test address update
-    await expect(store.getNanoContract('abc')).resolves.toMatchObject({ address: 'nanoAddress' });
-    await storage.updateNanoContractRegisteredAddress('abc', 'nanoAddress2');
-    await expect(store.getNanoContract('abc')).resolves.toMatchObject({ address: 'nanoAddress2' });
+    await expect(store.getNanoContract('abc')).resolves.toMatchObject({ address: address0.base58 });
+    await storage.updateNanoContractRegisteredAddress('abc', address1.base58);
+    await expect(store.getNanoContract('abc')).resolves.toMatchObject({ address: address1.base58 });
 
     // Go back to default address
-    await storage.updateNanoContractRegisteredAddress('abc', 'nanoAddress');
+    await storage.updateNanoContractRegisteredAddress('abc', address0.base58);
```

Comment:
Can we add another test that using `nanoAddress` fails with an error?

Reply:
Done https://github.com/HathorNetwork/hathor-wallet-lib/pull/657/commits/9cfca2b4cf24e858d7e6a0f8b5e19259c23e7dcd